services/iot-graphql-api/src/graphback.ts

```diff
@@ -0,0 +1,56 @@
+import { createKnexDbProvider } from '@graphback/runtime-knex'
+import { ApolloServer } from 'apollo-server-express'
+import { buildGraphbackAPI } from 'graphback'
+import { loadConfigSync } from 'graphql-config'
+import { removeNonSafeOperationsFilter, migrateDB } from 'graphql-migrations'
+import { connectDB, loadDBConfig } from './db'
+
+/**
+ * create Apollo server using automatically generated resolvers
+ */
+export const createApolloServer = () => {
+  const graphbackExtension = 'graphback'
+  const config = loadConfigSync({
+    extensions: [
+      () => ({
+        name: graphbackExtension
+      })
+    ]
+  })
+
+  const projectConfig = config.getDefault()
+  const graphbackConfig = projectConfig.extension(graphbackExtension)
+  const modelDefs = projectConfig.loadSchemaSync(graphbackConfig.model)
+
+  const db = connectDB()
+
+  const { typeDefs, resolvers, contextCreator } = buildGraphbackAPI(modelDefs, {
+    crud: {
+      create: true,
```

Comment:
```suggestion
      create: false,
```

Reply:
I missed this feature. Nice to see it exposed like this 👍

Reply:
Yep. We can enable them on models directly or globally. I felt like in this case global one is needed.